content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -170,6 +170,33 @@ kubelet instance. A new kubelet instance deletes all the existing Unix sockets u
 `/var/lib/kubelet/device-plugins` when it starts. A device plugin can monitor the deletion
 of its Unix socket and re-register itself upon such an event.
 
+### Device plugin and unhealthy devices
+
+There are cases when devices fail or being shut down. Device Plugin responsibility
+in this case is to notify kubelet about this situation using the `ListAndWatchResponse` API.
```

Comment:
I'd not be so specific in this form of documentation, I'd have generecally mentioned the device plugin interface and linked to it, but this is my personal taste and being specific is not wrong either IMO.

Reply:
I wanted to do more, but tight deadline is here. I will try to spend time later on further improvements